src/ngSanitize/sanitize.js

```diff
@@ -153,6 +154,42 @@ function $SanitizeProvider() {
       return buf.join('');
     };
   }];
+
+
+  /**
+   * @ngdoc method
+   * @name $sanitizeProvider#enableSvg
+   * @kind function
+   *
+   * @description
+   * Enables a subset of svg to be supported by the sanitizer.
+   *
+   * **Warning**: By enabling this setting without taking other precautions, you might expose your
```

Comment:
I would put this warning inside a `<div class="alert alert-warning">` block to make it stand out more.